internal/resources/aws/backup_vault.go

```diff
@@ -0,0 +1,163 @@
+
+	bd = backupData{ref: "monthly_efs_cold_restore_gb", name: "EFS restore (cold)", unit: "GB", usageType: "PartialRestore-Cold-EFS", service: "AWSBackup", family: "AWS Backup Storage"}
+	if r.MonthlyEFSColdRestoreGB != nil {
+		bd.qty = decimalPtr(decimal.NewFromFloat(*r.MonthlyEFSColdRestoreGB))
+	}
+	costComponents = append(costComponents, r.backupVaultCostComponent(bd))
+
+	bd = backupData{ref: "monthly_efs_item_restore_requests", name: "EFS restore (item-level)", unit: "requests", usageType: "PartialRestore-Jobs-EFS", service: "AWSBackup", family: "AWS Backup Storage"}
+	if r.MonthlyEFSItemRestoreRequests != nil {
+		bd.qty = decimalPtr(decimal.NewFromInt(*r.MonthlyEFSItemRestoreRequests))
+	}
+	costComponents = append(costComponents, r.backupVaultCostComponent(bd))
+
+	bd = backupData{ref: "monthly_ebs_snapshot_gb", name: "EBS snapshot", unit: "GB", usageType: "EBS:SnapshotUsage$", service: "AmazonEC2", family: "Storage Snapshot"}
+	if r.MonthlyEBSSnapshotGB != nil {
+		bd.qty = decimalPtr(decimal.NewFromFloat(*r.MonthlyEBSSnapshotGB))
+	}
+	costComponents = append(costComponents, r.backupVaultCostComponent(bd))
+
+	bd = backupData{ref: "monthly_rds_snapshot_gb", name: "RDS snapshot", unit: "GB", usageType: "RDS:ChargedBackupUsage", service: "AmazonRDS", family: "Storage Snapshot"}
+	if r.MonthlyRDSSnapshotGB != nil {
+		bd.qty = decimalPtr(decimal.NewFromFloat(*r.MonthlyRDSSnapshotGB))
+	}
+	costComponents = append(costComponents, r.backupVaultCostComponent(bd))
+
+	bd = backupData{ref: "monthly_dynamodb_backup_gb", name: "DynamoDB backup", unit: "GB", usageType: "TimedBackupStorage-ByteHrs", service: "AmazonDynamoDB", family: "Amazon DynamoDB On-Demand Backup Storage"}
+	if r.MonthlyDynamodbBackupGB != nil {
+		bd.qty = decimalPtr(decimal.NewFromFloat(*r.MonthlyDynamodbBackupGB))
+	}
+	costComponents = append(costComponents, r.backupVaultCostComponent(bd))
+
+	bd = backupData{ref: "monthly_dynamodb_restore_gb", name: "DynamoDB restore", unit: "GB", usageType: "RestoreDataSize-Bytes", service: "AmazonDynamoDB", family: "Amazon DynamoDB Restore Data Size"}
+	if r.MonthlyDynamodbRestoreGB != nil {
+		bd.qty = decimalPtr(decimal.NewFromFloat(*r.MonthlyDynamodbRestoreGB))
+	}
+	costComponents = append(costComponents, r.backupVaultCostComponent(bd))
+
+	bd = backupData{ref: "monthly_aurora_snapshot_gb", name: "Aurora snapshot", unit: "GB", usageType: "Aurora:BackupUsage", service: "AmazonRDS", family: "Storage Snapshot", key: "databaseEngine", value: "Aurora PostgreSQL"}
+	if r.MonthlyAuroraSnapshotGB != nil {
+		bd.qty = decimalPtr(decimal.NewFromFloat(*r.MonthlyAuroraSnapshotGB))
+	}
+	costComponents = append(costComponents, r.additionalBackupVaultCostComponent(bd))
+
+	bd = backupData{ref: "monthly_fsx_windows_backup_gb", name: "FSx for windows backup", unit: "GB", usageType: "BackupUsage", service: "AmazonFSx", family: "Storage", key: "fileSystemType", value: "Lustre"}
```

Comment:
```suggestion
	bd = backupData{ref: "monthly_fsx_windows_backup_gb", name: "FSx for Windows backup", unit: "GB", usageType: "BackupUsage", service: "AmazonFSx", family: "Storage", key: "fileSystemType", value: "Lustre"}
```
Realise this was wrong in the original code

Reply:
Fixed

Reply:
Fixed